<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Painter</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>631</width>
      <height>49</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSpinBox" name="spin_box_pen_size">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushbutton_ellipse">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>38</width>
       <height>38</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushbutton_ellipse {
background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
}

#pushbutton_ellipse:hover
{
border: 1px solid black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/oval-shape.png</normaloff>:/icons/icons/oval-shape.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_change_pen">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pb_change_pen{
background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
}

#pb_change_pen:hover
{
border: 1px solid black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/paint-palette.png</normaloff>:/icons/icons/paint-palette.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_brush">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_brush{
background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
}

#pushButton_brush:hover
{
border: 1px solid black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/paint-brush.png</normaloff>:/icons/icons/paint-brush.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushbutton_pen">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>38</width>
       <height>38</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushbutton_pen{
background-color: transparent;
background: none;
border: none;
}
#pushbutton_pen:hover
{
border: 1px solid black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/crayola.png</normaloff>:/icons/icons/crayola.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>23</width>
       <height>23</height>
      </size>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>16</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_brush">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushbutton_color">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>31</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushbutton_color{
background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
}

#pushbutton_color:hover
{
border: 1px solid black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/paint-pen.png</normaloff>:/icons/icons/paint-pen.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>16</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>16</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushbutton_resize">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushbutton_resize{
background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
}

#pushbutton_resize:hover
{
border: 1px solid black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/expand.png</normaloff>:/icons/icons/expand.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/icons/icons/pen24.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_pen">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid black;
background-color: black;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextEditable</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushbutton_line">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>31</width>
       <height>38</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushbutton_line {
background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
}

#pushbutton_line:hover
{
border: 1px solid black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/vector.png</normaloff>:/icons/icons/vector.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>19</width>
       <height>19</height>
      </size>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_shape">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>none</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>line</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>rect</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ellipse</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>pen</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushbutton_rect">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>38</width>
       <height>38</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushbutton_rect {
background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
}

#pushbutton_rect:hover
{
border: 1px solid black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/irectangle.png</normaloff>
       <selectedon>icons/rectangle_click.png</selectedon>:/icons/icons/irectangle.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushbutton_hand">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushbutton_hand{
background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
}

#pushbutton_hand:hover
{
border: 1px solid black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/hold.png</normaloff>:/icons/icons/hold.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_change_brush">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pb_change_brush{
background-color: transparent;
border-image: url(:off.png);
background: none;
border: none;
background-repeat: none;
}

#pb_change_brush:hover
{
border: 1px solid black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/icons/spray.png</normaloff>:/icons/icons/spray.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>590</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuHellp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDrawing">
    <property name="title">
     <string>Drawing</string>
    </property>
    <widget class="QMenu" name="menuShape">
     <property name="title">
      <string>Shape</string>
     </property>
     <addaction name="actionRectangle"/>
     <addaction name="actionEllipse"/>
     <addaction name="actionLine"/>
     <addaction name="actionText"/>
     <addaction name="actionPen"/>
    </widget>
    <addaction name="menuShape"/>
    <addaction name="actionPen_color"/>
    <addaction name="actionBrush_color"/>
    <addaction name="actionPen_size"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionMove"/>
    <addaction name="actionResize"/>
    <addaction name="actionChange_pen"/>
    <addaction name="actionChange_brush"/>
   </widget>
   <widget class="QMenu" name="menuNetworking">
    <property name="title">
     <string>Networking</string>
    </property>
    <addaction name="actionConnect_to_server"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuDrawing"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHellp"/>
   <addaction name="menuNetworking"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>121</red>
         <green>123</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>121</red>
         <green>123</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>121</red>
         <green>123</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>121</red>
         <green>123</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>121</red>
         <green>123</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>121</red>
         <green>123</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>121</red>
         <green>123</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>121</red>
         <green>123</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>121</red>
         <green>123</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(121, 123, 255);</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRectangle"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionLine"/>
   <addaction name="actionPen"/>
   <addaction name="actionText"/>
   <addaction name="separator"/>
   <addaction name="actionPen_color"/>
   <addaction name="actionBrush_color"/>
   <addaction name="actionPen_size"/>
   <addaction name="separator"/>
   <addaction name="actionMove"/>
   <addaction name="actionResize"/>
   <addaction name="separator"/>
   <addaction name="actionChange_pen"/>
   <addaction name="actionChange_brush"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/irectangle.png</normaloff>:/icons/icons/irectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/oval-shape.png</normaloff>:/icons/icons/oval-shape.png</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/vector.png</normaloff>:/icons/icons/vector.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/crayola.png</normaloff>:/icons/icons/crayola.png</iconset>
   </property>
   <property name="text">
    <string>Pen</string>
   </property>
  </action>
  <action name="actionText">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/text.png</normaloff>:/icons/icons/text.png</iconset>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="actionPen_color">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/paint-pen.png</normaloff>:/icons/icons/paint-pen.png</iconset>
   </property>
   <property name="text">
    <string>Pen color</string>
   </property>
  </action>
  <action name="actionBrush_color">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/paint-brush.png</normaloff>:/icons/icons/paint-brush.png</iconset>
   </property>
   <property name="text">
    <string>Brush color</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/hold.png</normaloff>:/icons/icons/hold.png</iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/expand.png</normaloff>:/icons/icons/expand.png</iconset>
   </property>
   <property name="text">
    <string>Resize</string>
   </property>
  </action>
  <action name="actionChange_pen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/paint-palette.png</normaloff>:/icons/icons/paint-palette.png</iconset>
   </property>
   <property name="text">
    <string>Change pen</string>
   </property>
  </action>
  <action name="actionChange_brush">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/spray.png</normaloff>:/icons/icons/spray.png</iconset>
   </property>
   <property name="text">
    <string>Change brush</string>
   </property>
  </action>
  <action name="actionPen_size">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/pen24.png</normaloff>:/icons/icons/pen24.png</iconset>
   </property>
   <property name="text">
    <string>Pen size</string>
   </property>
  </action>
  <action name="actionCurr_pen_color">
   <property name="text">
    <string>curr_pen_color</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/back.png</normaloff>:/icons/icons/back.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/forward.png</normaloff>:/icons/icons/forward.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionConnect_to_server">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connect to server</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
